src/Lib/LanguagesLib.php

```diff
@@ -690,6 +690,7 @@ public static function languagesInTatoeba()
                 'swc' => __d('languages', 'Congo Swahili'),
                 'rhg' => __d('languages', 'Rohingya'),
                 'mfa' => __d('languages', 'Kelantan-Pattani Malay'),
+                'skr' => __d('languages', 'Saraiki'),
```

Comment:
This did not actually fix the conflict. Although the content is the same, you're still modifying a line that was also modified in a parallel pull request. It's necessary to explicitly specify the intended result and mark the conflict as resolved.

GitHub has [specialized functionality for conflict resolution](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github). I think that should also be available to you as the author of the pull request. (If not, I can just resolve the conflict when merging.)

Reply:
I tried resolving it myself, let's see if it works.

Reply:
It did work (there's no conflict anymore), but it seems like you accidentally added a new empty line in the process.

Reply:
I don't see where the empty line is. If I leave it like that will it run a mistake?

Reply:
The empty line is number 695: https://github.com/Tatoeba/tatoeba2/pull/2987/commits/94e3e8bc1e7ea5d558b36d75e9469ffeecb1a5ab#diff-2f26e6cbce81128867f0a6ba0d187c4ff3b76872c5af503c4780dc9f54bcf02aR695

But it's not a big deal, so I'll merge this now.